### DISCUSSION TITLE
Package experience for new user?

### DISCUSSION THREAD

**Discussion by athowes** [epinowcast/epinowcast#490] [Announcements]
Has there been any user experience testing for people new to `epinowcast` and perhaps to nowcasting and epidemiological modelling in general?

As I'm just looking at the `README` I'm met with the first line as:

> Tools to enable flexible and efficient hierarchical nowcasting of right-truncated epidemiological time-series using a semi-mechanistic Bayesian model with support for a range of reporting and generative processes.

I think if you don't know what a lot of these terms mean it's going to be quite intimidating to use the package.

Perhaps it'd be good to get new users into the "Get started" vignette quicker? Rather than having to 1. read through a quite academic sounding abstract, 2. look over the folded texts about CmdStan and Docker, 3. open the folded text about vignettes.

As well, for the [vignettes/epinowcast.Rmd](https://github.com/epinowcast/epinowcast/blob/HEAD/vignettes/epinowcast.Rmd) I think this could be made more friendly to new users. As an example, the first figure in the vignette is all the way down when the model is fitted.

**Comment by seabbs** [comment]
There has been some but very little that is completely hands off (i.e with no developer interaction). I agree with all your points about surfacing a simple getting started (we used to have this in the README but it was ported to streamline things and perhaps this has gone too far).

**Comment by seabbs** [comment]
I'm going to move this to a discussion topic. @athowes if interested you could split your specific points into new issues?
